### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by shubham-singh-ind**

### CODE REVIEW

**Comment by rkaw92** [comment]
*shakes fist angrily at #hacktoberfest*

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # PM2 monitoring tool for Zabbix
 
-## What it is
+## What is it?
```
Comment (rootsher) [inline README.md:3]: should be burned to the ground!

Comment (Tuhaj) [reply README.md:3]: yes, that's exactly what is wrong with #Hacktoberfest 😢

Comment (withoutlogin) [reply README.md:3]: I think that's incompatible with this markdownlint rule: https://github.com/updownpress/markdown-lint/blob/master/rules/026-no-trailing-punctuation.md

**Pull request closed without merging**